astropy/modeling/core.py

```diff
@@ -2871,10 +3024,7 @@ def _prepare_outputs_single_model(model, outputs, format_info):
             else:
                 outputs[idx] = output.reshape(broadcast_shape)
 
-    if model.n_outputs == 1:
-        return outputs[0]
-    else:
-        return tuple(outputs)
+    return tuple(outputs)
```

Comment:
What is the reasoning of this change? It seems unrelated to unit support at a glance.